graph/change-tracking.md

```diff
@@ -0,0 +1,46 @@
+Change Tracking {align="center" style="text-align:center"}
+============
+
+Microsoft Graph API Design Pattern
+
+Change tracking enables users to keep a third party system in sync with changes in Microsoft Graph without having to continuously poll the API.
+
+Context
+-------
+
+In the world of distributed systems and richness of devices, keeping data in sync across systems is paramount to delivering a great user experience.
+
```

Comment:
In general i like it!  But i think we need to accommodate developers request to make it short and actionable thus  may be provide a concrete example like users or messages to describe the context then demonstrate a problem for API provider like how to enable frequent client querying without significant performance cost. Then solution would be like the workload must support a watermarks, API needs a function, annotations, etc. I created one pattern  as i see it, still need to do peer review though

Reply:
again most of the technical implementation aspects are already in delta.md.
As for the "architecture" pattern, the goal is to remove the need for clients to query often by providing them with an efficient to track changes.